package.json

```diff
@@ -131,21 +131,21 @@
     "@types/winreg": "^1.2.30",
     "@types/yauzl": "^2.9.1",
     "@types/yazl": "^2.4.2",
-    "@typescript-eslint/eslint-plugin": "3.2.0",
-    "@typescript-eslint/parser": "^3.3.0",
+    "@typescript-eslint/eslint-plugin": "4.15.1",
+    "@typescript-eslint/parser": "^4.15.1",
     "ansi-colors": "^3.2.3",
     "asar": "^3.0.3",
     "chromium-pickle-js": "^0.2.0",
-    "concurrently": "^5.2.0",
+    "concurrently": "^5.3.0",
     "copy-webpack-plugin": "^4.5.2",
     "cson-parser": "^1.3.3",
     "css-loader": "^3.2.0",
     "debounce": "^1.0.0",
     "deemon": "^1.4.0",
-    "electron": "9.4.3",
+    "electron": "11.2.3",
```

Comment:
We shouldn't be updating this unless we update the electron version we build with too @kburtram

Reply:
yeah, we can't update the electron version like this.  I have some pending vscode merge changes which bump electron, but otherwise we should change here.